test/unit/math/prim/scal/prob/bernoulli_test.cpp

```diff
@@ -6,37 +6,68 @@
 TEST(ProbDistributionsBernoulli, error_check) {
   boost::random::mt19937 rng;
   EXPECT_NO_THROW(stan::math::bernoulli_rng(0.6,rng));
+  EXPECT_NO_THROW(stan::math::bernoulli_logit_rng(-3.5,rng));
 
   EXPECT_THROW(stan::math::bernoulli_rng(1.6,rng),std::domain_error);
   EXPECT_THROW(stan::math::bernoulli_rng(-0.6,rng),std::domain_error);
   EXPECT_THROW(stan::math::bernoulli_rng(stan::math::positive_infinity(),rng),
                std::domain_error);
+  EXPECT_THROW(stan::math::bernoulli_logit_rng(stan::math::positive_infinity(),rng),
+               std::domain_error);
+}
+
+/*
+ * Uses a chi-squared test to check that counts are reasonably consistent
+ * with draws from a Bernoulli distribution..
+ */
+void assert_bernoulli(const std::vector<int>& counts, const std::vector<double>& expected_counts, double tolerance) {
```

Comment:
Happy to make a general chi square test, where in the tree do you think it should live?